docs/resources/mws_permission_assignment.md

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_mws_permission_assignment Resource
+
+These resources are invoked in the account context. Provider must have `account_id` attribute configured.
```

Comment:
+1 will there be a documentation for access/permission_assignment? For that one API doesn't need workspace id or account id, they can be derived from the workspace url.

Reply:
we don't have a way to retrieve account-level IDs from workspace in the provider yet. please "publish" the api ;)